contexts/deposit-context.tsx

```diff
@@ -119,7 +143,27 @@ export const DepositProvider = ({ children }: React.PropsWithChildren<{}>) => {
             },
           ],
         })
-        const { request } = await publicClient.simulateContract({
+        // const { request } = await publicClient.simulateContract({
```

Comment:
Revert back to using simulate after you're done debugging

Reply:
Done